sap-extensions/v2.0-extensions.json

```diff
@@ -50,6 +50,23 @@
             "upsert-collection",
             "action"
           ]
+        },
+        "x-sap-deprecated-operation": {
+          "type": "object",
+          "required": ["deprecationDate"],
+          "properties": {
+            "deprecationDate": {
+              "type": "string",
+              "format": "date",
+              "examples": ["2022-01-01", "2021-11-14"],
+              "description": "Deprecation date of an API operation (when the deprecation process started)."
+            },
+            "successorApi": {
+              "type": "string",
+              "format": "uri-reference",
+              "description": "Provides an URL to a successor API operation."
```

Comment:
How would you provide a URL to an API operation? There can be multiple API Operation for the same URLs. 

If we have a successor, I assume it would be a `successorOperation` within this API and we would point to the JSON Path of the operation.

Reply:
Yes, should work similar to the [`operationRef` field](https://spec.openapis.org/oas/v3.0.3#fixed-fields-16) in in Link Object.

Reply:
How about we allow either `operationRef` or `operationId` (mutually exclusive)? Then it is really the same as described in [Fixed Fields](https://spec.openapis.org/oas/v3.0.3#fixed-fields-16).